docs/layouts/docs.vue

```diff
@@ -79,6 +79,23 @@ export default {
       worker: false,
       cache: false,
     })
+
+    // adding event listener for resize
+    if(process.browser){
+      window.addEventListener("resize", this.onResize);
+    }
+
+    // this event checks for menu onclick
+    bus.$on('menuClicked', ()=> {
```

Comment:
we also need to unregister the event in destroyed life cycle

Reply:
Okay.